.changeset/khaki-crabs-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/date-time-input': patch
+---
+
+Update `moment` dependency to version 2.29.2
```

Comment:
Any reason for having multiple changesets? One is enough, since the changes are the same across these packages.